dev-libs/tree-sitter-haskell/tree-sitter-haskell-0_p20211114.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+TS_PV="d72f2e42c0d5ccf8e8b1c39e3642428317e8fe02"
```

Comment:
I might have missed it but I don't see anything checking TS_PV

Reply:
the eclass

Reply:
Ah yeah sorry, it already did that